### PR TITLE
Remove QUANTUM_SIZE from settings.js

### PR DESCRIPTION
**Pull request by sbc100**
This setting is already un-settable thanks for LEGACY_SETTINGS
system.

See #8257

### CODE REVIEW

File: src/library.js [hunk 1]
```diff
@@ -2248,9 +2248,9 @@ LibraryManager.library = {
     // report all function pointers as coming from this program itself XXX not really correct in any way
     var fname = stringToNewUTF8(Module['thisProgram'] || './this.program'); // XXX leak
     {{{ makeSetValue('info', 0, 'fname', 'i32') }}};
-    {{{ makeSetValue('info', QUANTUM_SIZE, '0', 'i32') }}};
-    {{{ makeSetValue('info', QUANTUM_SIZE*2, '0', 'i32') }}};
-    {{{ makeSetValue('info', QUANTUM_SIZE*3, '0', 'i32') }}};
+    {{{ makeSetValue('info', Runtime.QUANTUM_SIZE, '0', 'i32') }}};
+    {{{ makeSetValue('info', Runtime.QUANTUM_SIZE*2, '0', 'i32') }}};
+    {{{ makeSetValue('info', Runtime.QUANTUM_SIZE*3, '0', 'i32') }}};
```
Comment (kripken) [inline src/library.js]: We can probably remove that from Runtime too, and just hardcode `0, 4, 8` here. Although QUANTUM_SIZE might help fix things for wasm64 in the far future I guess.

Comment (sbc100) [reply src/library.js]: I'll do that as a followup.

Comment (dasa) [reply src/library.js]: Could this change be why when `EMCC_STRICT` is 1, I get this error:

```
/usr/local/Cellar/emscripten/1.38.31/libexec/src/compiler.js:189
Runtime.QUANTUM_SIZE = QUANTUM_SIZE;
                       ^

ReferenceError: QUANTUM_SIZE is not defined
    at Object.<anonymous> (/usr/local/Cellar/emscripten/1.38.31/libexec/src/compiler.js:189:24)
```

Comment (sbc100) [reply src/library.js]: Yes.  Seems likely .  I wonder why others haven't see this already.  Can you open a bug?

Comment (dasa) [reply src/library.js]: Thanks for #8555!

**Pull request merged**